app/services/usps_in_person_proofing/proofer.rb

```diff
@@ -16,23 +16,28 @@ def request_facilities(location)
         city: location.city,
         state: location.state,
         zipCode: location.zip_code,
-      }
+      }.to_json
 
-      resp = faraday.post(url, body, dynamic_headers)
+      headers = request_headers.merge(
+        'Authorization' => @token,
+        'RequestID' => request_id,
+      )
 
-      resp.body['postOffices'].map do |post_office|
-        PostOffice.new(
-          distance: post_office['distance'],
-          address: post_office['streetAddress'],
-          city: post_office['city'],
-          phone: post_office['phone'],
-          name: post_office['name'],
-          zip_code: post_office['zip5'],
-          state: post_office['state'],
-        )
+      resp = faraday.post(url, body, headers)
+      if resp.success?
```

Comment:
This will throw an error instead of returning on non-200 responses b/c we're using [the Faraday `RaiseError` middleware](https://github.com/lostisland/faraday/blob/v1.10.0/docs/middleware/response/raise_error.md).

Reply:
Removed unnecessary error checking, tests still pass.